src/DeviceInterface/ExtensionInterface.js

```diff
@@ -57,13 +57,8 @@ class ExtensionInterface extends InterfacePrototype {
         return null
     }
 
-    removeAutofillUIFromPage () {
-        super.removeAutofillUIFromPage()
-        this.activeForm?.removeAllDecorations()
```

Comment:
I centralized removing all decoration and everything else, so we don't need this override anymore.

Reply:
love it.